kipoiseq/dataclasses.py

```diff
@@ -284,7 +287,7 @@ def slop(self, upstream=0, downstream=0, use_strand=True):
         obj._end += downstream
         return obj
 
-    def is_valid(self, chrom_len=math.inf):
+    def is_valid(self, chrom_len=np.iinfo("int").max):
```

Comment:
`int(math.inf - 1)` does not work, so I thought this would be a better option.
If you care about int64 being to small, we could alternatively add some dtype option or just hope that a chromosome is shorter than `9223372036854775807`.

What kind of bugs do you have in mind?